test

```diff
@@ -3,135 +3,154 @@ set -ex
 
 DOMAIN_SOCKET="$(pwd)/mina-indexer.sock"
 
-# Create temporary directory for blocks and things in a cross platform
-# compatible way
 TMPDIR=$(mktemp -d 2>/dev/null || mktemp -d -t 'mytmpdir')
 BLOCKS_DIR="${TMPDIR}/blocks"
 mkdir $BLOCKS_DIR
 
 cleanup() {
     echo "Exiting with $?"
-    rm -rf $TMPDIR
-}
-trap cleanup EXIT
-
-cargo build
-
-teardown() {
-    rm -rf $HOME/.mina-indexer/database || true
-    rm -rf $HOME/.mina-indexer/logs || true
+    rm -rf $TMPDIR 2>/dev/null
```

Comment:
Could be disastrous if $TMPDIR is a directory with a space character in any of the directory components. Recommend surrounding by double quotes. Safer still is "${TMPDIR}".